### PR TITLE
Promote Tariffs v1.15.0 to Pre-Production

### PR DESCRIPTION
**Pull request by proctorj**
Reverts InformedSolutions/JAQU-CAZ-Tariff-Service#373

### CODE REVIEW

File: .drone2.yml [hunk 1]
```diff
@@ -1997,6 +1997,7 @@ steps:
       - mkdir ~/.aws
       - echo -e "[profile target_acc]\nrole_arn = $AWS_ACCOUNT_ROLE\ncredential_source = Environment\nrole_session_name = AWSACCOUNT" > ~/.aws/config
       - python3 ./ci-cd-resources/subscribe_to_alert_topics_iac_restructure.py -e freeable-memory -e 4xx -e usage -e evictions -e csv -i prod -i tariffs ISSupport-CAZ@informed.com
+      - python3 ./ci-cd-resources/subscribe_to_alert_topics_iac_restructure.py -e freeable-memory -e 4xx -e usage -e evictions -e csv -i prod -i tariffs -i database -i rds -i storage caz.support@justaftermidnight.io
```
Comment (kingarj) [inline .drone2.yml:2000]: Are we happy to take in the extra alerting?

**Review by kingarj — COMMENTED** [review]
Only other thought is - are we happy that we won't need the corresponding change to the reporting schema in VCCS (as should only be needed post CAZ launch)?

File: .drone2.yml (hunk 1 above)
Comment (proctorj) [reply .drone2.yml:2000]: Yes - Alerting put in place for our 24/7 monitoring partner is intended in this release

**Comment by proctorj** [comment]
> Only other thought is - are we happy that we won't need the corresponding change to the reporting schema in VCCS (as should only be needed post CAZ launch)?

Happy for that to come through in the next release.

**Review by kingarj — APPROVED** [review]

**Pull request merged**